genrl/deep/common/logger.py

```diff
@@ -153,15 +155,17 @@ def __init__(self, logdir: str):
         os.makedirs(self.logdir, exist_ok=True)
         self.writer = SummaryWriter(logdir)
 
-    def write(self, kvs: Dict[str, Any]) -> None:
+    def write(self, kvs: Dict[str, Any], log_key: str = "timestep") -> None:
         """
         Add entry to logger
 
         :param kvs: Entries to be logged
+        :param log_key: Key plotted on x_axis
         :type kvs: dict
+        :type log_key: str
         """
         for key, value in kvs.items():
-            self.writer.add_scalar(key, value, kvs["Timestep"])
+            self.writer.add_scalar(key, value, kvs[log_key])
```

Comment:
`kvs[log_key]` should be changed to just `log_key` maybe? Check the travis build error.

Reply:
kvs[log_key] works fine when I run it locally though.

Reply:
Yeah sorry, this isn't the problem